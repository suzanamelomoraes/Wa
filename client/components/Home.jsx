import React from 'react'
import { Container, Header, Divider, Image, Grid, Segment } from 'semantic-ui-react'

const inlineText = <div >
  <Header textAlign='center' style={{ fontSize: '30px', textShadow: '1px 1px #000000', color: '#ffffff' }}>
    <Header.Content>
    Nou te rourou <br/>
    Naku te rourou <br/>
    kia ora ai te iwi
    </Header.Content>
  </Header>
  <Divider inverted />
  <Header textAlign='center' style={{ fontSize: '30px', textShadow: '1px 1px #000000', color: '#ffffff' }}>
    <Header.Content>
      With your food basket  <br/>
      and my food basket <br/>
      the people will thrive test
    </Header.Content>
  </Header>
</div>

const Home = () => {
  const centerStyle = {
    display: 'flex',
    justifyContent: 'center',
    alignItems: 'center',
    flexWrap: 'wrap'
  }
  return (
  <>
  <Container
    style={{ width: '100%',
      marginTop: 60
    }}
  >
    <Image
      bordered
      style={{
        poistion: 'absolute',
        width: '100%',
        height: 720,
        margin: 0,
        padding: 0,
        opacity: 0.5,
        // backgroundImage: `url(${'images/placeholder.jpg'})`,
        // backgroundImage: `url(${'images/tomatoes.jpg'})`,
        // backgroundImage: `url(${'images/hands.jpg'})`,
        // backgroundImage: `url(${'images/handHearts.jpg'})`,
        backgroundImage: `url(${'images/waitangi.jpg'})`,
        backgroundSize: 'cover',
        display: 'flex',
        justifyContent: 'center',
        alignItems: 'center',
        flexWrap: 'wrap'
      }}
    >
      {inlineText}
    </Image>
  </Container>

  <Divider inverted />

  <Header style={centerStyle} size='huge'> Getting started </Header>

  <Divider inverted />

    <Grid colums='equal' style={centerStyle}>

      <Grid.Column mobile={12} tablet={8} computer={4}>
        <Header as='h3' attached='top' textAlign='center' size='huge' style={{ backgroundColor: '#123b26', color: 'white' }} >
        Mahi
        </Header>
        <Container style={{ width: '100%' }}>
          <Image src='images/placeholder.jpg' fluid
            style={{ opacity: 0.5 }}
          />

        </Container>

        <Segment attached size='huge' textAlign='center'>
          <br></br>
          <p>
            in literal sense mahi is the act of working/perfomring, Wa uses Mahi as a concept of offering a task doing what the people need and helping the commuity hauiahauih
          </p>
        </Segment>
      </Grid.Column>

      <Grid.Column mobile={12} tablet={8} computer={4}>
        <Header as='h3' attached='top' textAlign='center' size='huge' style={{ backgroundColor: '#123b26', color: 'white' }} >
        Community
        </Header>

        <Container style={{ width: '100%',
          height: '100%'
        }}>
          <Image src='images/hands.jpg' fluid
            style={{ opacity: 0.5 }}
          />
          {/* Create a mahi to share with your community and select a mahi to help out.
          </Image> */}

        </Container>

        <Segment attached size='huge' textAlign='center'>
          <p>
            a is very communtiy based platform as it aims to build a strong connection with those around you, become more aware of the skills you have and what others have to offer
            Contribute to the community by helping complete mahi
          </p>
        </Segment>
      </Grid.Column>

      <Grid.Column mobile={12} tablet={8} computer={4}>
        <Header as='h3' attached='top' textAlign='center' size='huge' style={{ backgroundColor: '#123b26', color: 'white' }} >
        Exchange
        </Header>

        <Container style={{ width: '100%',
          height: '100%'
        }}>
          <Image src='images/tomatoes.jpg' fluid
            style={{ opacity: 0.5 }}
          />
          {/* Create a mahi to share with your community and select a mahi to help out.
          </Image> */}

        </Container>

        <Segment attached size='huge' textAlign='center'>
          <p>      
            using your time to do the things that really matter

            exchaning time to do what you want to and doing the things that you cant do

            by offering an alternative to apprecaite and empower the communities

            Once mahi has been completed you exchange Wa/time with
          </p>
        </Segment>
      </Grid.Column>
    </Grid>

  </>

  )
}

export default Home
