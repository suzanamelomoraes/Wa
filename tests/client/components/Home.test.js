import React from 'react'
import { shallow, mount } from 'enzyme'
import toJson from 'enzyme-to-json'
import Home from '../../../client/components/Home'

test('Testing for true is truthy', () => {
  expect(true).toBeTruthy()
})

describe('Test <Home /> component (Shallow)', () => {
  it('Renders our <Grid />', () => {
    const expected = '<Grid />'
    const wrapper = shallow(<Home />)
    const actual = wrapper.text()

    expect(actual).toMatch(expected)
  })
  it('Renders our <Header />', () => {
    const expected = '<Header />'
    const wrapper = shallow(<Home />)
    const actual = wrapper.text()

    expect(actual).toMatch(expected)
  })
  it('Renders our <Container />', () => {
    const expected = '<Container />'
    const wrapper = shallow(<Home />)
    const actual = wrapper.text()

    expect(actual).toMatch(expected)
  })
  it('render unchanged from previous snapshot', () => {
    const wrapper = mount(<Home />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
