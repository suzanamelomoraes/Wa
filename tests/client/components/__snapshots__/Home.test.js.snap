// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <Home /> component (Shallow) render unchanged from previous snapshot 1`] = `
<Home>
  <Container
    style={
      Object {
        "marginTop": 60,
        "width": "100%",
      }
    }
  >
    <div
      className="ui container"
      style={
        Object {
          "marginTop": 60,
          "width": "100%",
        }
      }
    >
      <Image
        as="img"
        bordered={true}
        style={
          Object {
            "alignItems": "center",
            "backgroundImage": "url(images/waitangi.jpg)",
            "backgroundSize": "cover",
            "display": "flex",
            "flexWrap": "wrap",
            "height": 720,
            "justifyContent": "center",
            "margin": 0,
            "opacity": 0.5,
            "padding": 0,
            "poistion": "absolute",
            "width": "100%",
          }
        }
        ui={true}
      >
        <div
          className="ui bordered image"
          style={
            Object {
              "alignItems": "center",
              "backgroundImage": "url(images/waitangi.jpg)",
              "backgroundSize": "cover",
              "display": "flex",
              "flexWrap": "wrap",
              "height": 720,
              "justifyContent": "center",
              "margin": 0,
              "opacity": 0.5,
              "padding": 0,
              "poistion": "absolute",
              "width": "100%",
            }
          }
        >
          <div>
            <Header
              style={
                Object {
                  "color": "#ffffff",
                  "fontSize": "30px",
                  "textShadow": "1px 1px #000000",
                }
              }
              textAlign="center"
            >
              <div
                className="ui center aligned header"
                style={
                  Object {
                    "color": "#ffffff",
                    "fontSize": "30px",
                    "textShadow": "1px 1px #000000",
                  }
                }
              >
                <HeaderContent>
                  <div
                    className="content"
                  >
                    Nou te rourou 
                    <br />
                    Naku te rourou 
                    <br />
                    kia ora ai te iwi
                  </div>
                </HeaderContent>
              </div>
            </Header>
            <Divider
              inverted={true}
            >
              <div
                className="ui inverted divider"
              />
            </Divider>
            <Header
              style={
                Object {
                  "color": "#ffffff",
                  "fontSize": "30px",
                  "textShadow": "1px 1px #000000",
                }
              }
              textAlign="center"
            >
              <div
                className="ui center aligned header"
                style={
                  Object {
                    "color": "#ffffff",
                    "fontSize": "30px",
                    "textShadow": "1px 1px #000000",
                  }
                }
              >
                <HeaderContent>
                  <div
                    className="content"
                  >
                    With your food basket  
                    <br />
                    and my food basket 
                    <br />
                    the people will thrive
                  </div>
                </HeaderContent>
              </div>
            </Header>
          </div>
        </div>
      </Image>
    </div>
  </Container>
  <Divider
    inverted={true}
  >
    <div
      className="ui inverted divider"
    />
  </Divider>
  <Header
    size="huge"
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexWrap": "wrap",
        "justifyContent": "center",
      }
    }
  >
    <div
      className="ui huge header"
      style={
        Object {
          "alignItems": "center",
          "display": "flex",
          "flexWrap": "wrap",
          "justifyContent": "center",
        }
      }
    >
       Getting Started 
    </div>
  </Header>
  <Divider
    inverted={true}
  >
    <div
      className="ui inverted divider"
    />
  </Divider>
  <Grid
    colums="equal"
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexWrap": "wrap",
        "justifyContent": "center",
      }
    }
  >
    <div
      className="ui grid"
      colums="equal"
      style={
        Object {
          "alignItems": "center",
          "display": "flex",
          "flexWrap": "wrap",
          "justifyContent": "center",
        }
      }
    >
      <GridColumn
        computer={4}
        mobile={12}
        tablet={8}
      >
        <div
          className="four wide computer twelve wide mobile eight wide tablet column"
        >
          <Header
            as="h3"
            attached="top"
            size="huge"
            style={
              Object {
                "backgroundColor": "#123b26",
                "color": "white",
              }
            }
            textAlign="center"
          >
            <h3
              className="ui huge top attached center aligned header"
              style={
                Object {
                  "backgroundColor": "#123b26",
                  "color": "white",
                }
              }
            >
              Mahi
            </h3>
          </Header>
          <Container>
            <div
              className="ui container"
            >
              <Image
                as="img"
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundImage": "url(images/placeholder.jpg)",
                    "backgroundRepeat": "no-repeat",
                    "backgroundSize": "cover",
                    "display": "flex",
                    "flexWrap": "wrap",
                    "height": "250px",
                    "justifyContent": "center",
                    "opacity": 0.5,
                    "width": "100%",
                  }
                }
                ui={true}
              >
                <div
                  className="ui image"
                  style={
                    Object {
                      "alignItems": "center",
                      "backgroundImage": "url(images/placeholder.jpg)",
                      "backgroundRepeat": "no-repeat",
                      "backgroundSize": "cover",
                      "display": "flex",
                      "flexWrap": "wrap",
                      "height": "250px",
                      "justifyContent": "center",
                      "opacity": 0.5,
                      "width": "100%",
                    }
                  }
                >
                  <div>
                    <Header
                      style={
                        Object {
                          "color": "#ffffff",
                          "fontSize": "30px",
                          "textShadow": "1px 1px #000000",
                        }
                      }
                      textAlign="center"
                    >
                      <div
                        className="ui center aligned header"
                        style={
                          Object {
                            "color": "#ffffff",
                            "fontSize": "30px",
                            "textShadow": "1px 1px #000000",
                          }
                        }
                      >
                        <HeaderContent>
                          <div
                            className="content"
                          >
                            taking and offering mahi
                          </div>
                        </HeaderContent>
                      </div>
                    </Header>
                  </div>
                </div>
              </Image>
            </div>
          </Container>
          <Segment
            size="huge"
            textAlign="center"
          >
            <div
              className="ui huge center aligned segment"
            >
              <p>
                In literal sense mahi is the act of working or doing, Wa uses Mahi as a concept of offering a task. we need 5 lines here bruh.  we need 5 lines here bruh.  we need 5 lines here bruh.
              </p>
            </div>
          </Segment>
        </div>
      </GridColumn>
      <GridColumn
        computer={4}
        mobile={12}
        tablet={8}
      >
        <div
          className="four wide computer twelve wide mobile eight wide tablet column"
        >
          <Header
            as="h3"
            attached="top"
            size="huge"
            style={
              Object {
                "backgroundColor": "#123b26",
                "color": "white",
              }
            }
            textAlign="center"
          >
            <h3
              className="ui huge top attached center aligned header"
              style={
                Object {
                  "backgroundColor": "#123b26",
                  "color": "white",
                }
              }
            >
              Community
            </h3>
          </Header>
          <Container>
            <div
              className="ui container"
            >
              <Image
                as="img"
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundImage": "url(images/hands.jpg)",
                    "backgroundRepeat": "no-repeat",
                    "backgroundSize": "cover",
                    "display": "flex",
                    "flexWrap": "wrap",
                    "height": "250px",
                    "justifyContent": "center",
                    "opacity": 0.5,
                    "width": "100%",
                  }
                }
                ui={true}
              >
                <div
                  className="ui image"
                  style={
                    Object {
                      "alignItems": "center",
                      "backgroundImage": "url(images/hands.jpg)",
                      "backgroundRepeat": "no-repeat",
                      "backgroundSize": "cover",
                      "display": "flex",
                      "flexWrap": "wrap",
                      "height": "250px",
                      "justifyContent": "center",
                      "opacity": 0.5,
                      "width": "100%",
                    }
                  }
                >
                  <div>
                    <Header
                      style={
                        Object {
                          "color": "#ffffff",
                          "fontSize": "30px",
                          "textShadow": "1px 1px #000000",
                        }
                      }
                      textAlign="center"
                    >
                      <div
                        className="ui center aligned header"
                        style={
                          Object {
                            "color": "#ffffff",
                            "fontSize": "30px",
                            "textShadow": "1px 1px #000000",
                          }
                        }
                      >
                        <HeaderContent>
                          <div
                            className="content"
                          >
                            empowering communities
                          </div>
                        </HeaderContent>
                      </div>
                    </Header>
                  </div>
                </div>
              </Image>
            </div>
          </Container>
          <Segment
            size="huge"
            textAlign="center"
          >
            <div
              className="ui huge center aligned segment"
            >
              <p>
                communtiy based platform that aims to build a strong connection with those around you, become more aware of the skills you have and what others have to offer
              </p>
            </div>
          </Segment>
        </div>
      </GridColumn>
      <GridColumn
        computer={4}
        mobile={12}
        tablet={8}
      >
        <div
          className="four wide computer twelve wide mobile eight wide tablet column"
        >
          <Header
            as="h3"
            attached="top"
            size="huge"
            style={
              Object {
                "backgroundColor": "#123b26",
                "color": "white",
              }
            }
            textAlign="center"
          >
            <h3
              className="ui huge top attached center aligned header"
              style={
                Object {
                  "backgroundColor": "#123b26",
                  "color": "white",
                }
              }
            >
              Exchange
            </h3>
          </Header>
          <Container>
            <div
              className="ui container"
            >
              <Image
                as="img"
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundImage": "url(images/tomatoes.jpg)",
                    "backgroundRepeat": "no-repeat",
                    "backgroundSize": "cover",
                    "display": "flex",
                    "flexWrap": "wrap",
                    "height": "250px",
                    "justifyContent": "center",
                    "opacity": 0.5,
                    "width": "100%",
                  }
                }
                ui={true}
              >
                <div
                  className="ui image"
                  style={
                    Object {
                      "alignItems": "center",
                      "backgroundImage": "url(images/tomatoes.jpg)",
                      "backgroundRepeat": "no-repeat",
                      "backgroundSize": "cover",
                      "display": "flex",
                      "flexWrap": "wrap",
                      "height": "250px",
                      "justifyContent": "center",
                      "opacity": 0.5,
                      "width": "100%",
                    }
                  }
                >
                  <div>
                    <Header
                      style={
                        Object {
                          "color": "#ffffff",
                          "fontSize": "30px",
                          "textShadow": "1px 1px #000000",
                        }
                      }
                      textAlign="center"
                    >
                      <div
                        className="ui center aligned header"
                        style={
                          Object {
                            "color": "#ffffff",
                            "fontSize": "30px",
                            "textShadow": "1px 1px #000000",
                          }
                        }
                      >
                        <HeaderContent>
                          <div
                            className="content"
                          >
                            Using Wa/Time as a currency
                          </div>
                        </HeaderContent>
                      </div>
                    </Header>
                  </div>
                </div>
              </Image>
            </div>
          </Container>
          <Segment
            size="huge"
            textAlign="center"
          >
            <div
              className="ui huge center aligned segment"
            >
              <p>
                using your time to do the things that really matter time to do what you want to and doing the things that you cant do Once mahi has been completed you exchange Wa/time with
              </p>
            </div>
          </Segment>
        </div>
      </GridColumn>
    </div>
  </Grid>
</Home>
`;
